src/client/renderer/r_mesh_model.c

```diff
@@ -31,7 +31,11 @@ static void R_LoadMeshMaterial(r_model_t *mod) {
 	Dirname(mod->media.name, skin);
 	strcat(skin, "skin");
 
-	mod->mesh->material = R_LoadMaterial(skin);
+	cm_material_t *cm = Cm_LoadMaterial(skin);
+
+	mod->mesh->material = R_LoadMaterial(cm);
+
+	Cm_UnrefMaterial(cm);
```

Comment:
Seems to me like a little R_LoadMaterial wrapper which does this unref would be useful. I think this is the 3rd time I've seen the "grab, load, unref" triplet :)